packages/cli/openapi-parser/src/schema/examples/ExampleTypeFactory.ts

```diff
@@ -287,8 +287,23 @@ export class ExampleTypeFactory {
                             itemExamples.push(itemExample);
                         }
                     }
-                } else {
+                    // Otherwise, use a schema level example
+                } else if (schema.example != null && schema.example.length > 0) {
```

Comment:
if the full example doesn't contain the array example, then we default to a field specific example